docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -173,6 +173,195 @@ It is also possible to configure the scope values that would be used via the sco
+
+According to the https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests[OpenID Connect specifications],
+the `request` and the `request_uri` parameters enable Authentication Requests to be signed and optionally encrypted.
+Both of these parameters are optional and requests sent using these parameters are represented as a JWT. The
+`elytron-oidc-client` subsystem allows users to specify if they want to send their authentication requests as a JWT using
+the `autnentication-request-format` attribute. The `authentication-request-format`, `request-object-signing-algorithm`,
+`request-object-encryption-alg-value`, `request-object-encryption-enc-value` attributes are all available under
+the `secure-server`, `secure-deployment`, `realm` and `provider` resources.
+
+When configuring the server using the `elytron-oidc-client`, if the user wants all requests within a realm to be sent
+following the same format, then they can configure it inside the `realm` settings. If the user wants each
+deployment to have different authentication formats, then they can configure that using the `secure-server` or using the
+`secure-deployment` resources.
+
+The `authentication-request-format` attribute has values of type `String` and has 3 allowed values:
+
+* `oauth2`: This sends the authentication request as usual following the oauth2 syntax. The request parameters, such as,
+`state` are added to the authentication request as plain text. This is the default value for this attribute.
+* `request`: This sends the authentication request as a signed and optionally encrypted JWT, which the OpenID provider
+can verify and/or decrypt. The OpenID provider must support request parameters for the request to be communicated successfully. If the
+OpenID provider supports request parameter, the `request_parameter_supported` attribute should be set to `true`. If this
+value is set to `false`, then the request is sent using OAuth format and the corresponding message is displayed on the
+server log.
+* `request_uri`: This also sends the authentication request as a signed and optionally encrypted JWT, but `request_uri`
+sends the JWT by reference, as opposed to by value. The OpenID provider must support the `request_uri` parameter for the
+request to be communicated successfully. If the OpenID provider supports request parameter, the
+`request_uri_parameter_supported` attribute should be set to `true`. If this value is set to `false`, then the request
+is sent using OAuth format and the corresponding message is displayed on the server logs.
+
+NOTE: The default value for `authentication-request-format` is `oauth2`. If the OpenID provider does not support request
+parameters, then the request will be sent as usual following te OAuth2 format.
+
+=== Passing Request Object by Value
+
+In order to pass the JWT by value, the value for `authentication-request-format` must be set to `request`. According to
```

Comment:
Might be good to reword the first sentence to mention that it's possible to pass request objects by value when securing apps with OIDC.